.travis.yml

```diff
@@ -52,6 +52,7 @@ install:
   - pip install dist/*.tar.gz
   - pip install nose hacking mock
   - pip install autopep8
+  - pip install pynvrtc
```

Comment:
If `pynvrtc` is mandatory, don't install it here. Instead fix setup.py. And i feel you already fixed setup.py.